run_tool_shed.sh

```diff
@@ -32,10 +30,13 @@ if [ -z "$TOOL_SHED_CONFIG_FILE" ]; then
         TOOL_SHED_CONFIG_FILE=tool_shed_wsgi.ini
     elif [ -f config/tool_shed.ini ]; then
         TOOL_SHED_CONFIG_FILE=config/tool_shed.ini
+    elif [ -f config/tool_shed.yml ]; then
+        TOOL_SHED_CONFIG_FILE=config/tool_shed.yml
     else
-        TOOL_SHED_CONFIG_FILE=config/tool_shed.ini.sample
+        TOOL_SHED_CONFIG_FILE=config/tool_shed.yml.sample
     fi
     export TOOL_SHED_CONFIG_FILE
 fi
 
-python ./scripts/paster.py serve $TOOL_SHED_CONFIG_FILE --pid-file=tool_shed_webapp.pid --log-file=tool_shed_webapp.log $args
+find_server $TOOL_SHED_CONFIG_FILE
+$run_server $server_args
```

Comment:
@martenson @jmchilton After these changes we are setting but not using any more the `$args` variable, is this breaking the TS bootstrap?

Reply:
I doubt I tested this so I don't know, no one has complained though 😅. I wouldn't worry too much about that functionality unless someone really intends to use it. I just double checked a fresh Planemo build and its bootstrapped toolshed is working - but it is probably called in a fairly custom way (xref https://github.com/galaxyproject/ansible-galaxy-toolshed/blob/d1355719f431f9228952d57c6c73b47394dba07f/tasks/database.yml#L29).